qemu/tests/qsd_vubp_options.py

```diff
@@ -0,0 +1,88 @@
+"""QSD vhost-user-blk-pci device options test"""
+
+from provider.qsd import QsdDaemonDev, add_vubp_into_boot
+from virttest import error_context, env_process, utils_disk, utils_misc
+
+
+@error_context.context_aware
```

Comment:
no error_context.context() statement in codes, I think we can remove it. Others LGTM.

 (1/6) Host_RHEL.m9.u2.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.x86_64.io-github-autotest-qemu.qsd_vubp_options.with_mix_aim_off.q35: PASS (75.67 s)
 (2/6) Host_RHEL.m9.u2.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.x86_64.io-github-autotest-qemu.qsd_vubp_options.with_mix_aim_on.q35: PASS (73.56 s)
 (3/6) Host_RHEL.m9.u2.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.x86_64.io-github-autotest-qemu.qsd_vubp_options.with_packed.q35: PASS (72.98 s)
 (4/6) Host_RHEL.m9.u2.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win11.x86_64.io-github-autotest-qemu.qsd_vubp_options.with_mix_aim_off.q35: PASS (164.98 s)
 (5/6) Host_RHEL.m9.u2.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win11.x86_64.io-github-autotest-qemu.qsd_vubp_options.with_mix_aim_on.q35: PASS (166.20 s)
 (6/6) Host_RHEL.m9.u2.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win11.x86_64.io-github-autotest-qemu.qsd_vubp_options.with_packed.q35: PASS (161.65 s)

Reply:
updated